src/forwarders/file.js

```diff
@@ -32,7 +32,7 @@ function send (directory, data, type, separator, callback) {
     try {
         fs.writeFile(
             path.join(directory, 'boomcatch-' + uuid.v4() + '.' + extensions[type || 'default']),
-            data,
+            data + '\n',
```

Comment:
This is common in *nix, please let me know if I should use `require('os').EOL` instead.